test/Microsoft.Dnx.Tooling.FunctionalTests/DnuPackTests.cs

```diff
@@ -6,575 +6,596 @@
 using System.IO;
 using System.Linq;
 using Microsoft.AspNet.Testing.xunit;
-using Microsoft.Dnx.CommonTestUtils;
+using Microsoft.Dnx.Testing;
+using Newtonsoft.Json.Linq;
 using NuGet;
 using Xunit;
 
 namespace Microsoft.Dnx.Tooling
 {
-    public class DnuPackTests
+    public class DnuPackTests : DnxSdkFunctionalTestBase
```

Comment:
So I'm thinking we don't generate projects on the fly anymore. We can use the framework to put more projects in misc for each of the scenarios we care about but not during the test itself.